h2o-genmodel/src/main/java/hex/genmodel/ModelMojoReader.java

```diff
@@ -296,4 +296,8 @@ private void checkMaxSupportedMojoVersion() throws IOException {
       throw new IOException(String.format("MOJO version incompatibility - the model MOJO version (%.2f) is higher than the current h2o version (%s) supports. Please, use the older version of h2o to load MOJO model.", _model._mojo_version, mojoVersion()));
     }
   }
+
+  public MojoReaderBackend getMojoReaderBackend() {
```

Comment:
Shouldn't be exposed, this is too low level to be exposed to classes that inherit from the ModelMojoReader.

Reply:
@michalkurka current modifier for `_reader` is `protected`, so it was exposed for classes that inherit from MMR before my changes.  I should have made my method `protected` as well for not to break original encapsulation.  

Anyway I think that I need this `BufferedReader` from `MojoReaderBackend` so that I can parse encodings in one go.... like we do in `ModelMojoReader.parseModelInfo()`.  It is possible to use `readText` method but I will have to iterate through resulting `ArrayList` one more time to fill up EncodingMaps structure. 

Another approach could be is to store encodings in a separate files...but I will still have to iterate through ArrayList again to split `value` into numerator and denominator. And also I will have to use names of files(name of categorical column) as keys for encoding maps or store names of te columns separately in the model's description(which could inflate it in case of huge number of categorical columns).

So, wdyt? Do you still suggest to rewrite `TargetEncoderMojoReader.parseEncodingMap()` with  `readText`?